python/lesson1/tutorial.md

```diff
@@ -1,135 +1,114 @@
 ---
 layout: page
-title: Introduction to Python
+title: Installation Guide
 ---
 
-Welcome to the first Python tutorial. We're going to look at getting Python up and running on your computer, then look at a few simple operations to get to grips with the development environment, the syntax and basic computation.
+Welcome to the first Python tutorial. We're going to look at getting Python up
+and running on your computer, then look at a few simple operations to get to
+grips with the development environment and syntax.
 
 ## Installation
 
-On Mac, you'll already have Python installed.
+### Mac
 
-On Windows:
+Mac OSX ships with Python 2.7 which, while still a great language, is very
+outdated these days, so we need to upgrade:
+
+1. Make sure you're running an updated version of OSX.  You should at least
+   have OS 10.3, and most modern Macs are running much later than that anyway,
+   but it's worth noting this requirement here.
+2. Head to [the Python website](https://www.python.org/).
```

Comment:
Is it worth suggesting Homebrew instead? I know a bunch of the students already have it.